### PR TITLE
[Triton][docs] Added NVIDIA Triton CPU image

### PR DESCRIPTION
**Pull request by judyheflin**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 
### Description
- Added NVIDIA Triton Inference CPU image

## PR Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by maaquib — APPROVED** [review]

File: available_images.md [hunk 1]
```diff
@@ -135,6 +135,7 @@ NVIDIA Triton Inference Containers (SM support only)
 | Framework         |Job Type	|Horovod Options|CPU/GPU 	|Python Version Options	|Example URL																						|
 |-------------------|-----------|---------------|-----------|-----------------------|---------------------------------------------------------------------------------------------------|
 |NVIDIA Triton Inference Server 21.08    |inference	|No			|GPU 		| 3.8 (py38)			|007439368137.dkr.ecr.us-east-2.amazonaws.com/sagemaker-tritonserver:21.08-py3		|
+|NVIDIA Triton Inference Server 21.08    |inference	|No			|CPU 		| 3.8 (py38)			|007439368137.dkr.ecr.us-east-2.amazonaws.com/sagemaker-tritonserver:21.08-py3-cpu		|
```
Comment (saimidu) [inline available_images.md:138]: So just to confirm, `21.08-py3-cpu` leads to the CPU image, but `21.08-py3` leads to the GPU image?

Comment (saimidu) [inline available_images.md:138]: So just to confirm, `21.08-py3-cpu` leads to the CPU image, but `21.08-py3` leads to the GPU image?

**Review by saimidu — APPROVED** [review]

File: available_images.md (hunk 1 above)
Comment (maaquib) [reply available_images.md:138]: `21.08-py3` is both cpu and gpu. `21.08-py3-cpu` is a stripped down version with only cpu support

**Pull request merged**